setup.py

```diff
@@ -35,7 +35,7 @@ def find_version(*paths):
     packages=setuptools.find_packages(where=".", include="deltacat*"),
     install_requires=[
         # any changes here should also be reflected in requirements.txt
-        "boto3 == 1.20.24",
+        "boto3 == 1.20.40",
```

Comment:
Was this kept intentionally? When testing with Python 3.7 on my laptop, `pip install` fails to resolve compatible dependencies when `boto3 == 1.20.40`.

Reply:
```suggestion
        "boto3 == 1.20.24",
```